### DISCUSSION TITLE
[BUG] dropping out with "Killed"

### DISCUSSION THREAD

**Discussion by Netspud2K** [Chia-Network/chia-blockchain#5468] [Support]
**Describe the bug**
I think this is a bug. Happened twice.

I am executing: 
chia plots create -k 32 -b 3000 -n 20 -t /mnt/3tb/chiatmp1 -2 /mnt/3tb/chiatmp2 -d /mnt/3tb/chiaplots -e -r 3

Last few lines on the screen are: 
        Bucket 17 uniform sort. Ram: 2.866GiB, u_sort min: 1.375GiB, qs min: 0.687GiB.
        Bucket 18 uniform sort. Ram: 2.866GiB, u_sort min: 1.375GiB, qs min: 0.687GiB.
        Bucket 19 uniform sort. Ram: 2.866GiB, u_sort min: 1.375GiB, qs min: 0.687GiB.
        Bucket 20 uniform sort. Ram: 2.866GiB, u_sort min: 2.750GiB, qs min: 0.688GiB.
Killed 

Happened twice. Nothing in the debug.log -e is required or it crashes.

**To Reproduce**

chia plots create -k 32 -b 3000 -n 20 -t /mnt/3tb/chiatmp1 -2 /mnt/3tb/chiatmp2 -d /mnt/3tb/chiaplots -e -r 3

Wait hours, (it's a slower computer). It happens each time.

**Expected behavior**
A plot to be created.

**Screenshots**

**Desktop (please complete the following information):**

Distributor ID: Ubuntu
Description:    Ubuntu 20.04.2
Release:        20.04
Codename:       focal

model name      : Intel(R) Atom(TM) CPU N2800   @ 1.86GHz
RAM : 4Gb

**Additional context**

Guessing it's hardware related, as I have other plotters working just fine, some using a smaller buffer I think.

If you need more info left me know, and what you need mew to do to get it. It will take probably over a day to get the error again.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
